dynawo/sources/Modeler/DataInterface/IIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -1325,6 +1325,12 @@ DataInterfaceIIDM::getStaticParameterBoolValue(const std::string& staticID, cons
   return findComponent(staticID)->getStaticParameterValue<bool>(refOrigName);
 }
 
+boost::optional<std::string>
+DataInterfaceIIDM::getSlackNodeBusId() const {
```

Comment:
Shouldn't this go into the network API?

Reply:
For me it was a global API for data interface to be the most generic possible. Should it be in NetworkInterface ?

Reply:
@marcochiaramello any opinion?

Reply:
I think the concept of slacknode is useful regardless the data format, so I'd prefer it in NetworkInterface